news/2019-11-19-osu-idol-2019-finals-community-voting.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: post
+title: "osu!idol 2019 Finals - Community Voting!"
+date: 2019-11-19 12:00:00 +0000
+---
+
+It has been another amazing year for our osu!idol competition! There are three finalists we have for you to enjoy - [aceredshirt13](https://osu.ppy.sh/users/13736238), [moof](https://osu.ppy.sh/users/8457202), and [iamALieee](https://osu.ppy.sh/users/10093980) - go give them a listen and vote for your favourite!
+
+![](/wiki/shared/news/2019-11-19-osu-idol-2019-finals-community-voting/banner.jpg)
+
+### What is osu!idol?
+
+osu!idol is a singing contest driven by the community where contestants can participate solo or as a duet and sing their way through multiple stages that consist of different themes and genres to see who will reach the top! Now it is up to you to decide who will be the winner!
+
+[Last year's voting](https://osu.ppy.sh/home/news/2018-11-05-osu-idol-2018-finals-community-voting-results) resulted in the community choosing [Will Stetson](https://osu.ppy.sh/users/4909088) to be the osu!idol of 2018 with [Zeva[FL]](https://osu.ppy.sh/users/2805170) and [MjIsMaster](https://osu.ppy.sh/users/9922896) taking second and third place respectively. 
+
+Who will win this year? It is up to you!
+
+### Where can I vote?
+
+[Find the voting page here!](https://osu.ppy.sh/community/contests/83)
```

Comment:
this can be another sentence in the previous paragraph (and the header isn't necessary)

Reply:
removed header and moved to previous paragraph